instructor/migrations/0021_fixed_live_histogram.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('instructor', '0020_facs_sample_prep'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='flowcytometryhistogram',
+            old_name='data',
+            new_name='fixed_data',
+        ),
```

Comment:
question: is this part of the migration (`migrations.RenameField `) correct or it is just the result of django interpreting in the wrong way the new code?